### DISCUSSION TITLE
is there a different version of this extension for closing tab instead undo close tab?

### DISCUSSION THREAD

**Discussion by eldon922** [M-Reimer/undoclosetab#127] [Q&A] [answered]

**Comment by M-Reimer** [comment, marked as answer]
Not that I know of. And actually there are easy ways to close a tab without a new extension for this.

Keybard: Use the "Ctrl + W" shortcut to close a tab
Mouse: Middle-Click the tab, you want to close (and yes, pressing the mouse wheel is a "middle click").
